### ISSUE TITLE
TouchID not in Foucs

### ISSUE THREAD

**Issue by intdel** [keepassium/KeePassium#370]
**Description**
When using KeePassium on macOS Sonoma 14.5 with an AppCode and TouchID, the TouchID window is not in focus when opening the application.

**How to reproduce**
1. Enable AppCode with TouchID
2. Quit Application
3. Open Applicatio
4. TouchID is not in focus and putting finger on TouchID does nothing
5. Click on TouchID Window and everything works

**Expected behavior**
TouchID is in focus and I can unlock the app without having to put focus on TouchID window first

**Screenshots**
<img width="1040" alt="image" src="https://github.com/keepassium/KeePassium/assets/8001528/35f2a70e-41bf-480d-a09f-0d3f80ae00cb">
**Environment:**
 - Device: MacBook Pro 14
 - OS: macOS Sonoma 14.5
 - App Version: 1.51.147

**Comment by keepassium** [comment]
Yes, this happens. Not sure if this is a macOS thing or a Mac Catalyst quirk. Unfortunately, KeePassium has no control over that popup.

We'll be exploring a non-popup alternative shortly, hopefully it works out :)

**Comment by stukey** [comment]
Just migrated to Keepassium and would really love to see this fixed. It's a minor issue but with big impact and one of those little niggles that annoys you every time you use the app on macOS. Any update on when this will be fixed? Many thanks.
